worldedit-core/src/main/java/com/sk89q/worldedit/internal/cui/ServerCUIHandler.java

```diff
@@ -112,8 +113,14 @@ public static BaseBlock createStructureBlock(Player player) {
             return null;
         }
 
-        if (width > 32 || length > 32 || height > 32) {
-            // Structure blocks have a limit of 32x32x32
+        int dataVersion = WorldEdit.getInstance().getPlatformManager().queryCapability(Capability.GAME_HOOKS).getDataVersion();
+
+        // 1.16 increased maxSize to 48.
+        int maxSize = dataVersion >= 2566 ? 48 : 32;
```

Comment:
Can we extract this logic somewhere? Feels awkward to have it repeated twice, and if it gets expanded in the future we'd need to remember both places too.